cmd/main.go

```diff
@@ -137,7 +138,6 @@ var Options struct {
 	ValidationsConfig                    validations.Config
 	ManifestsGeneratorConfig             network.Config
 	UploaderConfig                       uploader.Config
-	EnableKubeAPI                        bool `envconfig:"ENABLE_KUBE_API" default:"false"`
```

Comment:
why did you made this change as part of this pr?

Reply:
I moved it to common.Config, so it is together with the soft timeouts configuration.
This gives the ability to call the function `Config.IsSoftTimeoutsEnabled` which needs both of these variables.

Reply:
why not make `ENABLE_SOFT_TIMEOUTS` with a default false and then enable it only in SAAS?

Reply:
I wanted to distinguish between cases that it is not set, it is set to false, or set to true.
If it will be default false, and enable it in SAAS, there will be no way to disable in SAAS.

Reply:
why not? If there is a bug that we want to avoid until this feature is stable we should be able to disable no?

Reply:
But this is your suggestion

Reply:
If the default is "false", there will not be a way to distinguish between when the variable value is received as default value and it is explicitly set to "false".  Therefore for SAAS, since we don't know to distinguish between these 2 cases, we should always set the value to true, which means that for SAAS it can never be disabled.